### PR TITLE
[documentation] #232: Add doc comments for filters

### PR DESCRIPTION
**Pull request by outoftardis**
Signed-off-by: Ekaterina Mekhnetsova <mekkatya@gmail.com>

### CODE REVIEW

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/transaction/Filters.kt [hunk 1]
```diff
@@ -91,8 +111,15 @@ object Filters {
...
 }
 
+/**
+ * `EntityFilters` match events associated with a given entity
+ * and either return all matching events or additionally apply another filter to them.
+ */
 object EntityFilters {
 
+    /**
+     * Match events associated with asset definition and apply [idFilter] or [eventFilter]
```
Comment (Mingela) [inline modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/transaction/Filters.kt:121]: I'd rephrase artificial parameter names to be included as human readable stuff in the docs, e.g. filter referenced by its [id][IdFilterAssetDefinitionId]

Comment (outoftardis) [reply modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/transaction/Filters.kt]: Updated

**Review by Mingela — APPROVED** [review]

**Review by arndey — APPROVED** [review]
